api-reference/10 UI Components/dxHtmlEditor/3 Methods/getBounds(index_length).md

```diff
@@ -3,16 +3,16 @@ id: dxHtmlEditor.getBounds(index, length)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Retrieves the pixel position and size of a selection at a specified location.
```

Comment:
As far as I understand, the method returns an object with `index` and `length`. How does this designate pixel position?